README.md

```diff
@@ -4,11 +4,9 @@ This demo includes all of the files necessary to get started with a basic, hello
 
 ## App Installation
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/bigcommerce/sample-app-nodejs)
-
 To get the app running locally, follow these instructions:
 
-1. [Use Node 14+ and NPM 7+](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#checking-your-version-of-npm-and-node-js)
+1. [Use Node 18.16 and NPM 9.5.1](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#checking-your-version-of-npm-and-node-js)
```

Comment:
I would specify use node 18+, rather than a specific version - this repo should support node 18, 19, and 20, as was as still supporting node versions that are currently in maintenance, e.g. 16

 ```suggestion
1. [Use Node 16+ and NPM 8+](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#checking-your-version-of-npm-and-node-js)
```